python/cuml/metrics/regression.pyx

```diff
@@ -97,3 +100,93 @@ def r2_score(y, y_hat, convert_dtype=False, handle=None):
     del y_m2
 
     return result
+
+
+@with_cupy_rmm
+def mean_squared_error(y_true, y_pred,
+                       sample_weight=None,
+                       multioutput='uniform_average', squared=True,
+                       handle=None):
+    """Mean squared error regression loss
+
+    Parameters
+    ----------
+    y_true : array-like (device or host) shape = (n_samples,)
+        or (n_samples, n_outputs)
+        Ground truth (correct) target values.
+    y_pred : array-like (device or host) shape = (n_samples,)
+        or (n_samples, n_outputs)
+        Estimated target values.
+    sample_weight : array-like (device or host) shape = (n_samples,), optional
+        Sample weights.
+    multioutput : string in ['raw_values', 'uniform_average']
+        or array-like of shape (n_outputs)
+        Defines aggregating of multiple output values.
+        Array-like value defines weights used to average errors.
+        'raw_values' :
+            Returns a full set of errors in case of multioutput input.
+        'uniform_average' :
+            Errors of all outputs are averaged with uniform weight.
+    squared : boolean value, optional (default = True)
+        If True returns MSE value, if False returns RMSE value.
+    handle : cuml.Handle, optional
```

Comment:
The cuml handle currently only applies to our algorithms that need to interface with C++, since the wrapped CUDA streams objects are separate between cupy and cuML. We have plans to create a cupy implementation of the cuML handle, but it has not been done yet.

Reply:
Right, good catch!